modules/profile/manifests/basenode.pp

```diff
@@ -1,26 +1,32 @@
 class profile::basenode {
 
   include motd
-  include base
+
+# schedule parameters for update
+  class { 'base':
+    period       => 'daily',
+    repeat       => '1',
+  }
 
 # Configure ssh_user
+# Defaulf user if083 password if083
   base::ssh_user { 'if083':
     ssh_user     => 'if083',
-    ssh_password => 'twm',
-    ssh_group    => 'wheel',
+    ssh_password => 'K7xRO54jYI7Ws',
```

Comment:
Now everyone who have access to our git repo (whole  world) can access our infrastructure.
Someone is already mining BitCoins
We all are fired.
Thanks!